deployments/dockerfiles/spotbugs/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM openjdk:8-jdk-alpine
+
+RUN apk add --no-cache curl tar bash wget unzip
```

Comment:
Would you mind replacing this RUN command into this new one: 

```

RUN apk update && apk upgrade \
	&& apk add --no-cache curl tar bash wget unzip openssh-client \
	&& apk add git
``` 

We need some other tools installed in this container as well. 🙃